scripts/l10n.ts

```diff
@@ -284,6 +287,63 @@ async function generateReport(missingTranslations: MissingTranslations) {
   return lines.join('\n');
 }
 
+type DocumentationReportItem = {
+  languageTag: string;
+  importName: string;
+  localeName: string;
+  missingKeysNb: number;
+  totalKeysNb: number;
+  githubLink: string;
+};
+const generateDocReport = async (
+  missingTranslations: MissingTranslations,
+  baseTranslationsNumber,
+) => {
+  const workspaceRoot = path.resolve(__dirname, '../');
+
+  packagesWithL10n.forEach(async ({ key: packageKey, documentationReportPath }) => {
+    const documentationReport: DocumentationReportItem[] = [];
+    Object.entries(missingTranslations).forEach(([importName, infoPerPackage]) => {
+      const info = infoPerPackage[packageKey];
+      if (info == null) {
+        return;
+      }
+      const githubLink = `${SOURCE_CODE_REPO}/blob/master/${info.path}/`;
```

Comment:
Maybe we should link to `next` in the next branch?
Or does the script only process master branch translations? 🤔 
Maybe we could just rely on `SOURCE_CODE_ROOT_URL` instead? 🤔

Reply:
Oh, gotcha. Sounds fair. 👌